scripts/apply_migration.py

```
@@ -0,0 +1,371 @@
+        logger.error(f"Migration failed: {e}")
+        return 1
+    except KeyboardInterrupt:
+        logger.info("Migration cancelled by user")
+        return 1
+    except Exception as e:
+        logger.error(f"Unexpected error: {e}")
+        return 1
+
+
+if __name__ == "__main__":
+    sys.exit(main())
```

Flake8 (7.2.0)

[warning] no newline at end of file

(W292)